source/uwp/Renderer/lib/AdaptiveActionEventArgs.h

```diff
@@ -3,18 +3,20 @@
 
 namespace AdaptiveCards { namespace XamlCardRenderer
 {
-    class AdaptiveSubmitActionEventArgs :
+    class AdaptiveActionEventArgs :
         public Microsoft::WRL::RuntimeClass<
         Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::RuntimeClassType::WinRt>,
-        ABI::AdaptiveCards::XamlCardRenderer::IAdaptiveSubmitActionEventArgs,
         ABI::AdaptiveCards::XamlCardRenderer::IAdaptiveActionEventArgs>
     {
-        InspectableClass(RuntimeClass_AdaptiveCards_XamlCardRenderer_AdaptiveSubmitActionEventArgs, BaseTrust)
+        InspectableClass(RuntimeClass_AdaptiveCards_XamlCardRenderer_AdaptiveActionEventArgs, BaseTrust)
 
     public:
         HRESULT RuntimeClassInitialize();
-        IFACEMETHODIMP get_ActionType(_Out_ ABI::AdaptiveCards::XamlCardRenderer::ActionType* actionType);
-    };
+        HRESULT RuntimeClassInitialize(ABI::AdaptiveCards::XamlCardRenderer::IAdaptiveActionElement* action);
+
+        // IAdaptiveActionEventArgs
+        IFACEMETHODIMP get_Action(ABI::AdaptiveCards::XamlCardRenderer::IAdaptiveActionElement** action);
 
-    ActivatableClass(AdaptiveSubmitActionEventArgs);
+        Microsoft::WRL::ComPtr<ABI::AdaptiveCards::XamlCardRenderer::IAdaptiveActionElement> m_action;
```

Comment:
Nope.  Fixed.

---
In reply to: [117513390](https://github.com/Microsoft/AdaptiveCards/pull/351#discussion_r117513390) [](ancestors = 117513390)